scholia/app/templates/base.html

```diff
@@ -253,19 +253,19 @@
 	 /* English Wikipedia */
 	 if ('enwiki' in item.sitelinks) {
 	     var title = item.sitelinks.enwiki.title;
-	     var wikipediaApiUrl = 'https://en.wikipedia.org/w/api.php?' +
-				   'action=query&prop=extracts&exsentences=3&exlimit=1&exintro=1&' + 
-				   'explaintext=1&callback=?&format=json&titles=' +
+	     var wikipediaApiUrl = 'https://en.wikipedia.org/api/rest_v1/page/summary/' +
 				   encodeURIComponent(title);
 	     var wikipediaUrl = 'https://en.wikipedia.org/wiki/' + encodeURIComponent(title)
-	     
-	     $.getJSON(wikipediaApiUrl, function(data) {
-		 var pages = data.query.pages;
-		 var text = pages[Object.keys(pages)[0]].extract + " ... "
-		 var html = "(from the <a href=\"" + wikipediaUrl + "\">English Wikipedia</a>)";
-		 $("#intro").text(text).append(html);
-	     }).fail(function(d, textStatus, error) {
-		 console.error("getJSON failed, status: " + textStatus + ", error: "+error)
+
+	     var headers = new Headers({"Api-User-Agent": "https://github.com/WDscholia/scholia"});
```

Comment:
This is something we should use more often.

Reply:
This is mentioned in #1245 , chose to use "Api-User-Agent" as there [is a issue with the "User-Agent" header in Chrome](https://stackoverflow.com/questions/42815087/sending-a-custom-user-agent-string-along-with-my-headers-fetch).